src/it/resources/com/google/checkstyle/test/chapter5naming/rule521$packagenames/InputPackageBadName3.java

```diff
@@ -1,4 +1,5 @@
 package com.google.checkstyle.test.chapter5naming.rule521$packagenames;
-// violation above 'Package name .* must match pattern .*'
+
+// violation 2 lines above 'Package name .* must match pattern .*'
```

Comment:
this update to avoid `'//' should be separated from previous line. [EmptyLineSeparator]`

Reply:
yes